src/operator.js

```diff
@@ -105,7 +105,7 @@ module.exports = class MonoplasmaOperator {
         // await this.finalPlasma.storeBlock(blockNumber) // TODO: give a timestamp
         await this.watcher.plasma.storeBlock(blockNumber)
         const tr = await tx.wait(1)        // confirmations
-        debug(`Commit sent, receipt: ${JSON.stringify(tr)}`)
+        this.log(`Commit sent, receipt: ${JSON.stringify(tr)}`)
```

Comment:
Doesn't look like this was actually logging.

Reply:
heh whops, good catch